### ISSUE TITLE
Failed to connect gradio connection timed out

### ISSUE THREAD

**Issue by lol55675** [marcoppasini/musika#43]
So then i run "Finetune a Musika model" it runs normally but it freezes when opening gradio public link.

Using CPU...

Calculating total number of samples in data folder...
Found 22 total samples
Dataset is ready!
Checking if models are already available...
Models are available!
/usr/local/lib/python3.9/site-packages/keras/initializers/initializers_v2.py:120: UserWarning: The initializer HeUniform is unseeded and being called multiple times, which will return identical values  each time (even if the initializer is unseeded). Please update your code to provide a seed to the initializer, or avoid using the same initalizer instance more than once.
  warnings.warn(
Networks loaded from /content/musika/checkpoints/techno
Critic params: 20786689
Generator params: 15499530
IMPORTANT: You are using gradio version 3.3.1, however version 4.29.0 is available, please upgrade.
--------
--------------------------------
--------------------------------
--------------------------------
--------------------------------
--------------------------------
CLICK ON LINK BELOW TO OPEN GRADIO INTERFACE
Running on local URL:  http://127.0.0.1:7860/
*** Failed to connect to ec2.gradio.app:22: [Errno 110] Connection timed out
